cumulus/client/consensus/aura/src/equivocation_import_queue.rs

```diff
@@ -177,6 +182,14 @@ where
 								.unwrap_or_default()
 						});
 
+					self.authorities_tracker.import(&pre_header, &*self.client).map_err(|e| {
```

Comment:
I think it would be good to import the authority set below the equivocation check below. Chances are really slim, but if we see an excessive amount of blocks for the same slot, we shouldn't pull all the authority set changes into our authority cache.